### ISSUE TITLE
aria-atomic missing when aria-live=assertive is used

### ISSUE THREAD

**Issue by ermcox** [ableplayer/ableplayer#585]
The SiteImprove system has flagged an issue with our use of Able Player on Indiana University websites. It states when aria-live="assertive" is used that is also must be accompanied by an aria-atomic="true" attribute. This issue is found in the playback speed container: span class="able-speed" aria-live="assertive"

Please update to include this attribute.

**Comment by joedolson** [comment]
Thanks, @ermcox!
